packages/ui-server/routes/api/init.js

```diff
@@ -33,7 +33,19 @@ router.get("/", function(req, res, next) {
 
 	questioner.start({
 		action: "question",
-		question: "A dummy question for reference"
+		question: "Will your application have multiple bundles?"
+	}).then((answer) => {
+		return entryQuestions(questioner, answer);
+	}).then((entryOptions) => {
+		if (entryOptions !== "\"\"") {
+			config.setWebpackOption("webpackOptions", {
+				entry: entryOptions,
+			});
+		}
+		return questioner.question({
+			action: "question",
+			question: "Which folder will Which folder will your generated bundles be in? [default: dist]"
```

Comment:
duplication in the phrase here.

Reply:
I don't get what you want to say here

Reply:
**Wich folder will Which folder will** ... The begin of the phrase is duplicated

Reply:
Okay I got it now 😂